build.gradle

```diff
@@ -186,3 +186,26 @@ subprojects {
         archives javadocJar
     }
 }
+
+if (!org.gradle.internal.os.OperatingSystem.current().isWindows()) {
+    task checkCopyright(type: Exec) {
+        description = 'Run tools/check-copyright.sh'
+        group = 'Verification'
+
+        commandLine 'tools/check-copyright.sh'
```

Comment:
How about replacing this script with the `grgit` Gradle plugin plus some custom logic instead? That would also make it work on Windows.

Reply:
Turns out `grgit` does not offer a `grep` command (because jgit doesn't) 😞